src/utils/equals.py

```diff
@@ -0,0 +1,12 @@
+class Comparison:
```

Comment:
Could add tests for this? 


I don't think this will work at all, because you will need to iterate through all keys and compare their values.

Maybe adding tests may help to test if this really works.

Also can you be more consistent when creating a file? don't a class `Comparison` when the file name is `equals`

Also I woudl suggest to just add the function in the file, I don't think you need a class for this.

Let me know wht you think about all of these suggestions and let me know any question.

Reply:
I added the test, but i test only if the function return the expected value (True/false) depending if the entities are
equals or no. now how can i test if the function is taking to much time or not?

Reply:
also:
* what is the disadvantage of modify the __eq__ in all the entities?
* what is the real use to compare object vs object when you can compare objet.id/name vs object.id/name??